extendr-api/src/iter.rs

```diff
@@ -101,14 +101,26 @@ impl<T: Copy> Iterator for SliceIter<T> {
 pub type NamedListIter = std::iter::Zip<StrIter, ListIter>;
 
 /// Iterator over primitives in integer objects.
+#[deprecated = "Use Int instead"]
```

Comment:
Does it make sense to deprecate rather than delete outright? Have they ever even been published on crates.io?

Reply:
I'd rather delete while we have limited numbers of users.

I think they are on crates.io, but they were aliases of slice iterators without lifetime
protection.

Reply:
Which begs the question- when do we publish!

Reply:
> Which begs the question- when do we publish!

I think the right strategy is to go back to issue #68 and the corresponding milestone and make a list of all the things we feel need to be addressed before we publish. We're probably close, but we haven't had this conversation recently.

Reply:
This is called a "Gap" analysis in the new trendy idiom :)

Reply:
I feel the gap is closing.

What is needed is documentation and examples.